tdrs-backend/tdpservice/backends.py

```diff
@@ -0,0 +1,59 @@
+"""Storage backends available for use within tdpservice."""
+from django.conf import settings
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+class OverriddenCredentialsS3Storage(S3Boto3Storage):
+    """An S3 storage class that overrides default settings with explicit values.
+
+    This is needed because S3Boto3Storage does not support using different
+    credentials and regions between different S3 storage classes.
+    """
+
+    def get_default_settings(self):
```

Comment:
This override is needed to prevent initialization of the base storage class from using the built in settings for these values.